src/instrument.js

```diff
@@ -411,6 +411,10 @@ function unliftStore(liftedStore, liftReducer) {
  * Redux instrumentation store enhancer.
  */
 export default function instrument(monitorReducer = () => null, options = {}) {
+  if (options.maxAge && options.maxAge < 2) {
+    throw new Error('DevTools maxAge option may not be < 2.');
```

Comment:
Let’s change it to `DevTools.instrument({ maxAge }) option, if specified, may not be less than 2.`